api/src/main/java/org/eclipse/microprofile/config/spi/Converter.java

```diff
@@ -24,7 +24,7 @@
  * <p>A Converter can specify a {@link javax.annotation.Priority}.
```

Comment:
I agree, but I don't see how we get `@Priority` annotation into MicroProfile then. Which API artifact would bring it? In the current state, there's no way to build an app with `@Priority` with just MP dependencies.